ansible/roles/test/files/ptftests/arista.py

```diff
@@ -290,8 +290,13 @@ def extract_from_logs(self, regexp, data):
             m = re_compiled.match(line)
             if not m:
                 continue
-            raw_data.append((datetime.datetime.strptime(
-                m.group(1), "%b %d %X"), m.group(2), m.group(3)))
+            try:
+                raw_data.append((datetime.datetime.strptime(
```

Comment:
Can we always add the 'year' for parsing? it will make it more consistent.

Reply:
updated